src/stylesheets/core/_utilities.scss

```diff
@@ -191,7 +191,7 @@
     display: block;
     font-family: $font-sans;
     line-height: 1.3;
-    padding: 0.85rem 1rem 0.85rem 1.8rem;
+    padding: 0.85rem 1rem 0.85rem $site-margins-mobile;
```

Comment:
Is the left padding of `$site-margins-mobile` something we want across all sizes or only mobile screen? The current sidenav has a slightly larger left margin and want to make sure.

Reply:
Yes, let's leave this as it. I don't think the extra 3px do enough to justify deviation from the "standard" indent